### DISCUSSION TITLE
Automatic Voucher renewal

### DISCUSSION THREAD

**Discussion by JoseLuisTeixeira** [Art-of-WiFi/UniFi-API-client#198] [Q&A] [answered]
In our company, we are using Guest wifi to provide free wifi to all employees.

I found this code to create a voucher, which works well.
https://github.com/Art-of-WiFi/UniFi-API-client/blob/master/examples/create_voucher.php

I would like to understand how to use API to extend or renew the validity of a voucher / voucher list.

Can someone provide me some insight on how to achieve this?

**Comment by malle-pietje** [comment, marked as answer]
AFAIK there is no method available through the UniFi Hotspot Manager to modify existing vouchers. If there is one, we can verify which API route(s) that uses and then add support to the API Client class.
